srcpkgs/vala-panel/template

```diff
@@ -1,22 +1,29 @@
 # Template file for 'vala-panel'
 pkgname=vala-panel
-_glhash=da591ffc8460b2a473e1aba3e3dcf178
-version=0.5.0
+version=24.05
 revision=1
 build_style=meson
-configure_args="$(vopt_if wayland -Dplatforms=x11,layer-shell -Dplatforms=x11)"
+build_helper=gir
 hostmakedepends="vala pkg-config gettext glib-devel"
-makedepends="gtk+3-devel libwnck-devel $(vopt_if wayland gtk-layer-shell-devel)"
+makedepends="gtk+3-devel libwnck-devel gobject-introspection
+ $(vopt_if wayland gtk-layer-shell-devel)"
 short_desc="GTK+3 desktop panel forked from LXPanel"
 maintainer="Joshua Krämer <joshua@kraemer.link>"
 license="LGPL-3.0-or-later"
 homepage="https://gitlab.com/vala-panel-project/vala-panel"
-distfiles="https://gitlab.com/vala-panel-project/vala-panel/uploads/$_glhash/vala-panel-${version}.tar.xz"
-checksum=ee3a9e000133da43b218ecd1738463c2de3f45783a5a09a11b9c263df4e233d1
+distfiles="https://gitlab.com/vala-panel-project/vala-panel/-/archive/${version}/vala-panel-${version}.tar.bz2"
+checksum=adde420a1f3edd7fec947b06482ff3f2e65a7e1568bd0b0bd1ba39a051fbbdb5
 
 # Package build options
-build_options="wayland"
-build_options_default="wayland"
+build_options="x11 wayland"
+build_options_default="x11 wayland"
+if [ "$build_option_x11" -a "$build_option_wayland" ]; then
+	configure_args="-Dplatforms=x11,wayland"
+elif [ "$build_option_x11" ]; then
+	configure_args="-Dplatforms=x11"
+elif [ "$build_option_wayland" ]; then
+	configure_args="-Dplatforms=wayland"
+fi
```

Comment:
the x11 build option just adds a bunch of complexity, i would put it back to how it was